base/src/main/scala/data/IdentitySyntax.scala

```diff
@@ -0,0 +1,42 @@
+package scalaz
+package data
+
+import annotation.tailrec
+import scalaz.typeclass.{Applicative, Monoid}
+import language.implicitConversions
+
+final class IdOps[A](val self: A) extends AnyVal {
+  /** Applies the provided function to `self`. The Thrush combinator. */
+  def |>[B](f: A => B): B =
```

Comment:
If we use this, as opposed to, say, `<#>` or `&`, we should probably also think about `<|` where appropriate (not here).

Reply:
It would be defined on `FunctionOps`.

Reply:
Oh, `def <|(a: A): B = self.apply(a)`, not `tap` (like in scalaz 7)? That makes more sense.